### ISSUE TITLE
Always show root folder.

### ISSUE THREAD

**Issue by Auratelience** [notaapp/talk#274]
Hi Nota team. Would it be possible to enable always being able to see the root folder in a workspace?

Thanks. Aura

**Comment by astoilkov** [comment]
Yes, we can do that.

Currently, the folder is shown only when there are notes/images inside of the root. What's your use case? Why do you want to always see it?

Thanks!

**Comment by Auratelience** [comment]
I typically have a folder for each project that I am working on. I do try to have everything in a sub folder, but readmes or other markdown things that apply to the whole project I keep in the root folder.

**Comment by astoilkov** [comment]
Is it hard for you to create the first file in the root folder? Maybe we can add a new command in the menu "New Root Note" when the root isn't visible. Will that help?

I'm exploring alternatives because we hide the root as a convenience for other users. It's a thing of minimalism. That's why I'm searching for something that solves your problem while keeping the UI as clean as possible.

**Comment by Auratelience** [comment]
Yes! That would be great. I understand (and like) the minimalistic / clean interface. A command would work well.

**Comment by Auratelience** [comment]
One other thing that could be good is adding a checkbox in the workspace window to show root folder. That might be easier.
